src/scenes/CollectionGalleryPage/CollectionGalleryHeader.tsx

```diff
@@ -135,8 +130,14 @@ function CollectionGalleryHeader({
           <StyledBreadcrumbsWrapper>
             <StyledUsernameWrapper>
               <StyledUsernameAndSeparatorWrapper>
-                <StyledUsernameMobile onClick={handleBackClick}>{username}</StyledUsernameMobile>
-                {collection.name && <StyledSeparatorMobile>/</StyledSeparatorMobile>}
+                {username ? (
+                  <UnstyledLink href={`/${username}`}>
```

Comment:
Because this is a link to a profile/username, I didn't think it was necessary to pipe this through a `useBackButton` handler, but let me know if I missed something.

Reply:
ahh yeah we wanna keep the back functionality mainly bc if you're scrolled down halfway through the page, then click "back" to the username, you want to end up back where you were. in contrast, just directing them to `/username` puts them back at the top of the page

Reply:
i realize this is quite hacky

Reply:
We chatted about this over Discord and decided it's ~fine to not have "back button" behavior here. If we want to reintroduce this sort of behavior (thinking about a Twitter timeline where you might click into a tweet and hitting a back arrow/link ideally would take you to where you left off in your timeline view), we can do it in a more generic way rather than wiring it up to every click handler.